examples/function-appevent-transformation/javascript/example.js

```diff
@@ -0,0 +1,47 @@
+import { FunctionEventHandler as EventHandler } from '@contentful/node-apps-toolkit';
+import https from 'https';
+
+const appEventTransformationingHandler = (event, context) => {
+  // If event is an entry, get lat and long fields and geocode them
+  if (event.entityType === 'Entry') {
+    const lat = event.entityProps.fields.lat;
+    const long = event.entityProps.fields.long;
+    if (lat && long) {
+      const url = `https://maps.googleapis.com/maps/api/geocode/json?latlng=${lat},${long}&key=YOUR_API_KEY`;
+
+      https
+        .get(url, (response) => {
+          let data = '';
+
+          response.on('data', (chunk) => {
+            data += chunk;
+          });
+
+          response.on('end', () => {
+            const result = JSON.parse(data);
+            console.log(result);
+
+            if (result.results.length > 0) {
+              const address = result.results[0].formatted_address;
```

Comment:
should return something for this one, maybe just the address or maybe something like the event with the address tacked on